doc/manual/source/parameters/index.rst

```diff
@@ -951,10 +951,74 @@ Hierarchy Control Parameters
     refined to at all times. (No default setting)
 ``MustRefineRegionLeftEdge`` (external)
     Bottom-left corner of refinement region. Must be within the overall
-    refinement region. Default: 0.0 0.0 0.0
+    refinement region. If using a moving refinement region, this will
+    ideally correspond to the bottom-left corner in the
```

Comment:
I think the use if "ideally" here is a little misleading... Maybe "...this should correspond..." or "...this should be set to the bottom-left..."....

Reply:
Thanks, I'll fix that!